### ISSUE TITLE
Encoding corpus with method = "R" and more than p-attribute = "word" leads to crash

### ISSUE THREAD

**Issue by ChristophLeonhardt** [PolMine/cwbtools#24]
Beginning Version 0.1.0, encoding more than the p-attribute "word" apparently doesn't work when the method = "R". In this version, the `p_attribute_encode()` method was changed to use `cwb_makeall()` instead of the system command when the method is set to "R".

A reproducible example which leads to a fatal crash of R can be derived from the vignette you provided. For example in the Jane-Austen-Scenario, there is a p-attribute "stem" which can be added in the encoding step:

```{r, message = FALSE}
Austen$encode(
   corpus = "AUSTEN", encoding = "utf8",
   p_attributes = c("word", "stem"), s_attributes = "book",
   registry_dir = registry_tmp, data_dir = austen_data_dir_tmp,
   method = "R", compress = FALSE
)
```

While this can be circumvented by using method = "cwb", this isn't the default scenario in the documentation.

**Comment by ablaette** [comment]
The crash results from a sensitivity of `cwb_makeall()` to work correctly if the corpus is loaded. The corpus needs to be unloaded before a second, third ... nth p-attribute can be written. I have implemented this in a new version on  the dev branch.-

The vignette is adapted accordingly, i.e. the p-attribute "stem" is written in addition to "word". Furthermore, I introduced a unit test that downloads the CWB to a temporary directory and checks the result of the R method against the CWB method.
